music/serializers.py

```diff
@@ -11,13 +11,22 @@ class MusicSerializer(serializers.ModelSerializer):
     # Validators for post method
     name = serializers.CharField(max_length=255, required=True)
     url = serializers.CharField(max_length=512, required=True)
-    duration = serializers.IntegerField(required=True)
+    total_duration = serializers.IntegerField(required=True)
     room_id = serializers.IntegerField(required=True, write_only=True)
     source = serializers.CharField(required=True)
 
     class Meta:
         model = Music
-        fields = ('pk', 'music_id', 'name', 'thumbnail', 'count', 'duration', 'timer_start', 'timer_end', 'url', 'room_id', 'source', 'last_play')
+        fields = ('pk', 'music_id', 'name', 'thumbnail', 'count', 'duration', 'total_duration', 'timer_start', 'url', 'room_id', 'source', 'last_play')
+
+    def create(self, validated_data):
+        music = Music(**validated_data)
+        music.save()
+        return music
+
+    def update(self, instance, validated_data):
```

Comment:
Yep, each time we use Serializer.save(), if create or update are implemented, it override the normal save behavior.
I use it to do the same thing but return the Music object correspondant.
Else it return a MusicSerializer object.